.dockerignore

```diff
@@ -0,0 +1,10 @@
+__pycache__
```

Comment:
Can we also just add an ignore for PyCharm .idea

Reply:
Sure, will do so tomorrow morning.